docs-md/operate-and-deploy/installation/installing.md

```diff
@@ -3,144 +3,222 @@ layout: page
+- For Windows, use
+  [Docker Desktop for Windows](https://docs.docker.com/docker-for-windows/install/).
+  Change the **Memory** setting on the
+  [Advanced](https://docs.docker.com/docker-for-windows/#advanced) settings
+  page to 8 GB.
+- For Linux, follow the [instructions](https://docs.docker.com/install/)
+  for your Linux distribution. No memory change is necessary, because Docker
+  runs natively and not in a VM.
+
+Create a stack file to define your ksqlDB application 
+-----------------------------------------------------
+
+When you've decided on the services that you want in the stack, you define a
+[Compose file, or "stack" file](https://docs.docker.com/compose/compose-file/),
+which is a YAML file, to configure your ksqlDB application's services. The 
+stack file is frequently named `docker-compose.yml`. 
+
+To start your ksqlDB application, you use the
+[docker-compose CLI](https://docs.docker.com/compose/) to
+run the stack for your application. Run `docker-compose up` to start your
+application and `docker-compose down` to stop it.
+
+!!! note
+    If your stack file is compatible with version 3 or higher,
+    you can use the `docker stack deploy` command:
+    `docker stack deploy -c docker-compose.yml your-ksqldb-app`.
+    For more information, see
+    [docker stack deploy](https://docs.docker.com/engine/reference/commandline/stack_deploy/).
+
+Build a ksqlDB application
+--------------------------
+
+The following steps describe how to define and deploy a stack for a ksqlDB
+application.
+
+### 1.  Define the services for your ksqlDB application
+
+Decide which services you need for your ksqlDB application. 
+
+For a local installation, include one or more {{ site.ak }} brokers in the
+stack and one or more ksqlDB Server instances. 
+
+- {{ site.zk }} -- one, for cluster metadata
+- {{ site.ak }} -- one or more
+- {{ site.sr }} -- optional, but required for Avro
+- ksqlDB Server  -- one or more
+- ksqlDB CLI -- optional
+- Other services -- like Elasticsearch, optional
+
+!!! note
+    A stack that runs {{ site.sr }} can handle Avro-encoded events. Without
+    {{ site.sr }}, ksqlDB handles only JSON or delimited schemas for events. 
+
+You can declare a container for the ksqlDB CLI in the stack, or you can attach
+the CLI to a ksqlDB Server instance later, from a separate container.
+
+### 2.  Build the stack
+
+Build a stack of services and deploy them by using
+[Docker Compose](https://docs.docker.com/compose/).
+
+Define the configuration of your local ksqlDB installation by creating a
+[Compose file](https://docs.docker.com/compose/compose-file/), which by
+convention is named `docker-compose.yml`.
+
+### 3. Bring up the stack and run ksqlDB
+   
+To bring up the stack and run ksqlDB, use the
+[docker-compose](https://docs.docker.com/compose/reference/overview/) tool,
+which reads your `docker-compose.yml` file and runs containers for your
+{{ site.ak }} and ksqlDB services. 
+
+ksqlDB Tutorial stack
```

Comment:
curious why cap used in "Tutorial" here...notice you are using sentence style cap.